packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -39,6 +39,13 @@ const Duration _kExpand = Duration(milliseconds: 200);
 /// ** See code in examples/api/lib/material/expansion_tile/expansion_tile.0.dart **
 /// {@end-tool}
 ///
+/// {@tool dartpad}
+/// This sample shows the creation of a [ExpansionTile] using [ThemeData.useMaterial3] flag,
```

Comment:
Please replace this sentence with something like: This example demonstrates how the expansion icon's location and appearance can be customized.

Reply:
Updated original example with this better description.